test/jasmine/tests/gl_plot_interact_basic_test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+var Plotly = require('@lib/index');
+var Plots = require('@src/plots/plots');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var mouseEvent = require('../assets/mouse_event');
+
+function teardown(gd, done) {
+
+    // The teardown function needs information of what to tear down so afterEach can not be used without global vars.
+    // In addition to removing the plot from the DOM it also destroy possibly present 2D or 3D scenes
+
+    // TODO we should figure out something to only rely on public API calls
```

Comment:
Oh. `Plotly.purge` should do the trick. It was added back in https://github.com/plotly/plotly.js/pull/300